### ISSUE TITLE
Plotly figure cropped with JupyterLab 4.0.0a30

### ISSUE THREAD

**Issue by jtpio** [jupyterlab/jupyterlab#13227]
## Description

Reporting here since this is an issue with JupyterLab 4 but not with 3:

**4.0.0a30**

![image](https://user-images.githubusercontent.com/591645/195179036-e48ed7a9-ff37-4f7c-8f28-64dbcf30d015.png)

**3.4.7**

![image](https://user-images.githubusercontent.com/591645/195178001-87c91df9-8980-48af-b1fb-8039000bb9fe.png)

## Reproduce

1. `pip install plotly pandas`
2. Open JupyterLab
3. Run a cell with the following content:

```python
import plotly.express as px
fig = px.scatter(x=[0, 1, 2, 3, 4], y=[0, 1, 4, 9, 16])
fig.show()
```

## Expected behavior

The figure should be displayed correctly.

## Context

- Operating System and version: Fedora 36
- Browser and version: Firefox 105
- JupyterLab version: 4.0.0a30

cc @fcollonval who might know if that could be related to the virtual notebook changes

**Comment by fcollonval** [comment]
@jtpio does it resize after a certain delay or not? Because at first the plotly height is small as it displays here (I saw that in the integration tests).

**Comment by jtpio** [comment]
No it does not seem to resize, even when resizing the window or triggering changes:

https://user-images.githubusercontent.com/591645/195293840-8629a9cd-f5c2-46ec-bc57-374b25e13401.mp4

**Comment by fcollonval** [comment]
Thanks for the screencast.

**Comment by fcollonval** [comment]
I can confirm this is still happening on `master`

I think the reason is that the CSS rule applied is: 

> In CSS 2, the effect of [min-height](https://drafts.csswg.org/css2/#propdef-min-height) and [max-height](https://drafts.csswg.org/css2/#propdef-max-height) on tables, inline tables, table cells, table rows, and row groups is undefined.

coupled to the current set of CSS rule:

![image](https://user-images.githubusercontent.com/8435071/218148238-04f366af-1bf3-46ae-95ef-65e2f722146e.png)

Setting `height` instead of `min-height` works but I think it needs to be applied upstream.

---

ref: https://drafts.csswg.org/css2/#min-max-heights

The switch to table layout was done in https://github.com/jupyterlab/jupyterlab/pull/11508

**Comment by jtpio** [comment]
Update: this was fixed in https://github.com/plotly/plotly.py/pull/4227.

We can keep the issue open until there is a new release with the fix.

**Comment by jtpio** [comment]
Closing as it is now fixed with Plotly 5.15.0: https://github.com/plotly/plotly.py/releases/tag/v5.15.0

![image](https://github.com/jupyterlab/jupyterlab/assets/591645/86a82e74-478e-403a-bed4-fdab45f7e01b)
